### ISSUE TITLE
Remove a role from a group

### ISSUE THREAD

**Issue by justinehell** [ditrit/leto-modelizer-admin#80]
Blocked by #84 

On GroupPage, add an action on RolesTable.
Make actions optional and table don't do the result of action, only send the event that action are click on the parent.

You can call the props something like that: with-delete-action.

On delete event, display a confirmation popup and confirmation delete the group role association.

![image](https://github.com/ditrit/leto-modelizer-admin/assets/62500081/e712d748-b30f-463c-ba07-025d857ca1d8)

**Comment by justinehell** [comment]
Task put on hold, will be done when the new api is up.
